.gitignore

```diff
@@ -1,3 +1,9 @@
+## IDE
+*.iml
+.idea/*
+!/.idea/runConfigurations
+.vscode/
```

Comment:
Should we include the .vscode directory? It would be nice to have the file associations + JSON schema already hooked up.

Reply:
This is driven by `settings.json`. Here, I also have some other stuff defined by the Java extension. Same for `launch.json`. How about commiting a *light* version as `settings.json.dist` or something along those lines?